src/amo/components/GetFirefoxButton/index.js

```diff
@@ -67,7 +67,7 @@ export const GetFirefoxButtonBase = (props: InternalProps) => {
       );
       buttonText = i18n.gettext('Only with Firefox—Get Firefox Now');
       puffy = true;
-      utmContent = `rta:${_base64url.encode(addon.guid)}`;
+      utmContent = addon.guid ? `rta:${_base64url.encode(addon.guid)}` : '';
```

Comment:
I didn't realize that guid could be null. Does this only happen with search tool extensions? I am wondering if we should have something more descriptive than an empty string tho (or maybe keep it to what it was before (null) )..?

also, when I refresh the page (sometimes it's the 2nd refresh), I still see an error.  Do you see that too?

Reply:
> Does this only happen with search tool extensions? 

I think so.

> I am wondering if we should have something more descriptive than an empty string tho (or maybe keep it to what it was before (null) )..?

Empty string, just like `null` and `undefined` are not appended to the query string anyway, so it is safe and it is what was before the base64 patch.

> also, when I refresh the page (sometimes it's the 2nd refresh), I still see an error. Do you see that too?

Which error? I don't get any error related to this link :/

Reply:
> Empty string, just like null and undefined are not appended to the query string anyway, so it is safe and it is what was before the base64 patch.

👍 

> Which error? I don't get any error related to this link :/

I actually restarted the server and it seems fine now:)